### PR TITLE
Changed themes to high contrast boxes and added monokai theme

### PR DESCRIPTION
**Pull request by securisec**
image

### CODE REVIEW

File: libr/cons/d/cga [hunk 1]
```diff
@@ -40,7 +40,7 @@ ec help rgb:0ff
 ec trap rgb:000 rgb:f0f
 
 ec graph.box rgb:f0f
-ec graph.box2 rgb:f0f
+ec graph.box2 rgb:ff0
```
Comment (radare) [inline libr/cons/d/cga]: Yellow is not a valid cga color

File: libr/cons/d/consonance [hunk 2]
```diff
@@ -38,7 +38,7 @@ ec creg rgb:789
 ec flag rgb:4cf
 
 ec graph.box rgb:789
-ec graph.box2 rgb:d41
+ec graph.box2 rgb:d31ee8
```
Comment (radare) [inline libr/cons/d/consonance]: Use f0f.. its shorter and its magenta too

File: libr/cons/d/cutter [hunk 3]
```diff
@@ -33,7 +33,7 @@ ec flow2 rgb:8d0
 ec mov black
 ec invalid rgb:e33
 ec graph.box rgb:28d
-ec graph.box2 rgb:8d0
+ec graph.box2 rgb:ff0
```
Comment (radare) [inline libr/cons/d/cutter]: Dont think yellow brights too much on the white background of cutter

File: libr/cons/d/dark [hunk 4]
```diff
@@ -34,7 +34,7 @@ ec trap rgb:000 rgb:
 ec swi rgb:99a
 ec invalid rgb:rgb:44f rgb:123
 ec graph.box rgb:040
-ec graph.box2 rgb:040
+ec graph.box2 rgb:f218f2
```
Comment (radare) [inline libr/cons/d/dark]: Use 3 digit rgb and reuse and this color is magenta, why dont use another color from the same palete?

File: libr/cons/d/lima [hunk 5]
```diff
@@ -39,7 +39,7 @@ ec invalid rgb:ef0
 ec trap rgb:000 rgb:af2
 
 ec graph.box rgb:af2
-ec graph.box2 rgb:0a0
+ec graph.box2 rgb:0ef
```
Comment (radare) [inline libr/cons/d/lima]: Turqioise is not part of the lime color palette

File: libr/cons/d/matrix [hunk 6]
```diff
@@ -35,7 +35,7 @@ ec swi rgb:0b0
 ec creg rgb:080
 ec flag rgb:0f0
 ec graph.box green
-ec graph.box2 rgb:060
+ec graph.box2 rgb:0ef
```
Comment (radare) [inline libr/cons/d/matrix]: Turqoise is not matrix green

File: libr/cons/d/ogray [hunk 7]
```diff
@@ -38,7 +38,7 @@ ec graph.trufae rgb:ddd
 ec graph.current rgb:f72
 ec graph.traced rgb:f00
 ec graph.box rgb:f72
-ec graph.box2 rgb:f72
+ec graph.box2 rgb:ff0
```
Comment (radare) [inline libr/cons/d/ogray]: Yellow is not orange

File: libr/cons/d/pink [hunk 8]
```diff
@@ -38,7 +38,7 @@ ec graph.trufae rgb:f5d
 ec graph.current rgb:f5d
 ec graph.traced rgb:a7f
 ec graph.box rgb:f5d
-ec graph.box2 rgb:f5d
+ec graph.box2 red
```
Comment (radare) [inline libr/cons/d/pink]: Red is not pink but i guess its good

File: libr/cons/d/smyck [hunk 9]
```diff
@@ -37,7 +37,7 @@ ec ai.read rgb:66d
 ec ai.write rgb:d66
 ec ai.exec rgb:6d6
 ec graph.box rgb:9df
-ec graph.box2 rgb:9df
+ec graph.box2 rgb:602050
```
Comment (radare) [inline libr/cons/d/smyck]: This is not part of the smyck palette, use 3 digit

File: libr/cons/d/white [hunk 10]
```diff
@@ -33,7 +33,7 @@ ec flow2 green
 ec mov black
 ec invalid red
 ec graph.box blue
-ec graph.box2 green
+ec graph.box2 rgb:00fffa
```
Comment (radare) [inline libr/cons/d/white]: Use a basic color if possibel or 3 digit rgb

**Comment by securisec** [comment]
Ah. You are right, I did not consider the color pallets itself. If you want, you can take from this pull request what you would like, like the new theme (although it is it not fully monokai now that you mention it. it is smyck with integration of the main monokai colors). Or i can close this pull request for now and re sumit once i have a chance to pay a lot more attention to the themes.

**Comment by radare** [comment]
you can update this pr by comiting the fixes i commented.

or you can commit your new theme in a separate PR and then improve the others in another PR.

thanks!

**Comment by securisec** [comment]
Perfect! I created a new PR for the monokai theme, and will re open this once i make the fixes based on the comments. Thanks!

**Pull request closed without merging**